plugins/in_opentelemetry/opentelemetry_prot.c

```diff
@@ -332,6 +335,30 @@ static int otlp_pack_any_value(msgpack_packer *mp_pck,
     return result;
 }
 
+/*
```

Comment:
This comment is wrong because it alludes to the new log record format, please remove it.